lib/order.rb

```diff
@@ -0,0 +1,49 @@
+require 'csv'
+# Order is responsible for tracking orders with prices for all orders from CSV
+class Order
+  attr_accessor :products, :customer, :fulfillment_status
```

Comment:
These don't need to all be accessors, because your tests don't require the user is able to directly change `products`, `customer`, and `fulfillment_status` after the `Order` instance is created.